ts/components/SessionScrollButton.tsx

```diff
@@ -18,7 +17,10 @@ const SessionScrollButtonDiv = styled.div`
   }
 `;
 
-export const SessionScrollButton = (props: { onClickScrollBottom: Noop }) => {
+export const SessionScrollButton = (props: {
+  onClickScrollBottom: () => void;
+  unreadCount: number;
+}) => {
```

Comment:
```suggestion
export const SessionScrollButton = (props: {
  onClickScrollBottom: () => void;
  }) => {
 const unreadCount =  useSelectedUnreadCount();
```

Reply:
you can add this hook in `ts/state/selectors/selectedConversation` by copying one of the existing one like `useSelectedIsLeft()`